packages/react-dom/src/__tests__/ReactDOMComponent-test.js

```diff
@@ -1034,33 +1034,31 @@ describe('ReactDOMComponent', () => {
       return (str + '').replace(/([.?*+\^$\[\]\\(){}|-])/g, '\\$1');
     }
 
-    function toHaveAttribute(actual, expected) {
+    function expectToHaveAttribute(actual, expected) {
       const [attr, value] = expected;
       let re = '(?:^|\\s)' + attr + '=[\\\'"]';
       if (typeof value !== 'undefined') {
         re += quoteRegexp(value) + '[\\\'"]';
       }
-      return new RegExp(re).test(actual);
+      expect(new RegExp(re).test(actual)).toBe(true);
```

Comment:
this _could_ be `expect(actual).toMatch(new RegExp(re))` fwiw. If it fails, it'll print `actual` instead of `expected false to be true`

Reply:
Could you include this in https://github.com/facebook/react/pull/16332?

Reply:
Certainly! Will update tomorrow

Reply:
Done